.gitignore

```diff
@@ -2,3 +2,6 @@ MANIFEST
 .DS_Store
 _book
 node_modules
+.idea
+.swp
```

Comment:
Doesn't this need to be ```*.swp```?

Reply:
It can be `.*.swp` but I think that is alright.